index.js

```diff
@@ -6,6 +6,10 @@ var { DEFAULT_EXTENSIONS } = require('./constants');
 
 var rules = requireUp('eslint-local-rules', DEFAULT_EXTENSIONS, __dirname);
 
+if (!rules) {
+  rules = requireUp('eslint-local-rules', DEFAULT_EXTENSIONS, process.cwd());
+}
```

Comment:
@cletusw 
In workspaces, use `process.cwd()` to find the file if it is not found.
This should also work for workspaces.
what do you think about this?